### DISCUSSION TITLE
Any plan for har support?

### DISCUSSION THREAD

**Discussion by hirokith** [sindresorhus/got#1946] [Ideas]
#### What problem are you trying to solve?

Getting har support to export and view the request & response detail like https://github.com/exogen/node-fetch-har.

...

#### Describe the feature

Got support pass extra params to export the har json object for caller to produce the request detail view, like https://github.com/exogen/node-fetch-har.

...

#### Checklist

- [x] I have read the documentation and made sure this feature doesn't already exist.

**Comment by szmarczak** [comment]
It's rather out of our scope. [HAR has been abandoned by the W3C group](https://w3c.github.io/web-performance/specs/HAR/Overview.html), so this is unlikely to be done by us.

**Reply by hirokith** [reply]
What a pity...
